model/ids.go

```diff
@@ -145,6 +146,13 @@ func (t *TraceID) UnmarshalJSON(data []byte) error {
 	return t.Unmarshal(b)
 }
 
+func (t *TraceID) ToOTELTraceID() pcommon.TraceID {
```

Comment:
@yurishkuro I got this from https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/coreinternal/idutils/big_endian_converter.go#L13-L18. Is that fine? I can't reuse it directly because it lives in `internal`

Reply:
no problem, but it's good to leave a comment explaining that with a reference to the original code.